keyvault/client/src/main/java/com/azure/keyvault/KeyVaultErrorCodeStrings.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I think the suggestion was to create a property file with these strings similar to: https://docs.oracle.com/javase/tutorial/i18n/intro/steps.html

Reply:
Updated this file to read error strings from properties file defined under src/main/resources

Reply:
I wonder if there is value in adding a base class for reading error strings. For example, have all error strings stored in a default file name (ie. "errorstrings.properties") and you can modify your `load()` method to load resources depending on the class.